JamesNojekChallenge/Contact.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@page
+@model HeadstormFrontend.Pages.ContactModel
+@{
+    ViewData["Title"] = "Contact";
+}
+
+<h1>Contact</h1>
+
+<script src="https://www.google.com/recaptcha/api.js?render=6LenDZcaAAAAAMtYKn7gK8qYGKS0UCqrH1Le6ubI"></script> <!--used my own key to test recaptcha on localhost-->
+<!--recaptcha code from google-->
+<script>
+    function onSubmit(token) {
+        document.getElementById("demo-form").submit();
+    }
+</script>
+
+
+<!--gets data from variables to dump into console log-->

```

Comment:
Is there a reason you have the script tag in the middle of the file?

Reply:
Yes, lines 11-15 contain a script with a function called onSubmit that allows the console dump to occur when the button is clicked. I'm not sure if this is the best way to cause that event to happen, but it's what I found in my research